insights/parsers/sys_block.py

```diff
@@ -36,10 +38,16 @@ def parse_content(self, content):
             raise ParseException('Error: {0}'.format(content if content else 'empty file'))
         try:
             self._stable_writes = int(content[0].strip())
+            self._device = self.file_path.split('/')[3]
```

Comment:
Since the file_path would be a absolute path in the runtime, it would be better to use `[-3]` here instead of `[3]`

Reply:
Yes, good point.